### PR TITLE
Fix `0.get_idcard()` runtime with holoparasites

### PR DESCRIPTION
**Pull request by Absolucy**
## About The Pull Request

fixes a runtime. simple enough.

## Why It's Good For The Game

feex bug

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
fix: Fixed a runtime error related to holoparasites with ID cards.
/:cl:

### CODE REVIEW

**Review by EvilDragonfiend — APPROVED** [review]

**Comment by EvilDragonfiend** [comment]
The logic is obvious. No need to suggest the testing evidence.
but it needs 24 hours to be merged since the bug is low-prioritised, and it's not an exploit.

File: code/modules/holoparasite/_holoparasite.dm [hunk 1]
```diff
@@ -229,14 +229,14 @@ GLOBAL_LIST_EMPTY_TYPED(holoparasites, /mob/living/simple_animal/hostile/holopar
 		. += "<span data-component=\"RadarChart\" data-width=\"300\" data-height=\"300\" data-area-color=\"[accent_color]\" data-axes=\"Damage,Defense,Speed,Potential,Range\" data-stages=\"1,2,3,4,5\" data-values=\"[stats.damage],[stats.defense],[stats.speed],[stats.potential],[stats.range]\" />"
 
 /mob/living/simple_animal/hostile/holoparasite/get_idcard(hand_first = TRUE)
-	//Check hands
+	// IMPORTANT: don't use ?. for these, because held_item might be 0 for some reason!!
```
Comment (Kapu1178) [inline code/modules/holoparasite/_holoparasite.dm:232]: masking a bug. get_active_held_item() should never be returning a value that isnt null or an item instance

Comment (Absolucy) [reply code/modules/holoparasite/_holoparasite.dm:232]: well i don't feel like tracking that down

Comment (EvilDragonfiend) [reply code/modules/holoparasite/_holoparasite.dm:232]: Hmm, maybe that FALSE might have been a failsafe.
@Absolucy Would you post a testing evidence please?

Comment (Absolucy) [reply code/modules/holoparasite/_holoparasite.dm:232]: I don't even know what triggered the runtime, but sure in a bit

Comment (Absolucy) [reply code/modules/holoparasite/_holoparasite.dm:232]: @EvilDragonfiend here's it uh... not runtiming? 
23-11-29-1701285068-dreamseeker

**Review by EvilDragonfiend — APPROVED** [review]

**Pull request merged**